civictechprojects/static/css/_vars.scss

```diff
@@ -45,6 +28,7 @@ $color-brand-80: #f06d23;
 $color-neutral-10: #ffffff;
 $color-neutral-15: #f6f7f8;
 $color-neutral-20: #f5f6f6;
+$color-neutral-25: #f2f2f2;
 $color-neutral-30: #eaeef0;
 $color-neutral-45: #8c8c8c; // rgba(25,25,25,0.5) to rgb; may not be correct, waiting for design response 04/2022
```

Comment:
$color-neutral-45
#191919
Opacity 50%
rgb (25, 25, 25)

For the overlay screen. Or I can copy the current overlay HEX/RGB to the DS, so you don't need to change it on the site. But it will be corrected in Figma.

Reply:
If the opacity is essential I can do it here. I'll just set it to read `$color-neutral-45: rgba(25,25,25,0.5); // for overlay only`

Reply:
It is for the screen when a modal dialog is open. I believe the current is $color-opacity-layer: #231f20; //  overlay opacity. What percentage of opacity does it give? I can change $color-neutral-45 to #231f20 instead of #191919 but I need to know the percentage :)

Reply:
Ah, I see. I did some investigation and `color-opacity-layer` isn't used for modals, it's used for splash images -- like the homepage's top image + buttons.

Instead, our modals use Bootstrap's default `.modal-backdrop` which is currently #000000, opacity 0.5; I can change that to rgba(25, 25, 25, 0.5) if you like.

Reply:
ok, then change to rgba(25, 25, 25, 0.5), please. Then, we won't use a true black color #00000 anywhere on the site. Thanks!

Reply:
Turns out Bootstrap requires us to declare the rgb and the opacity separately so I declared it as #191919 with a big note saying 'never use without opacity 0.5' -- best I can do, I'm afraid.